### ISSUE TITLE
Editor Welcome: overlay controls are hidden on large touch devices

### ISSUE THREAD

**Issue by razvanpapadopol** [Automattic/wp-calypso#55821]
### What
Currently, welcome tour overlay controls (minimize and close buttons) are hidden on devices with a viewport width larger than 480px and displayed only when hovered with a mouse. We should either show the controls everywhere or target the touch-enabled devices instead of mobile resolutions to make them visible.

### Why
Users should be able to control the editor welcome tour on touch-enabled devices.

### Screenshot
<img width="500" alt="Screenshot 2021-08-30 at 10 52 51" src="https://user-images.githubusercontent.com/14192054/131305570-184b584f-4aeb-47d6-9f8a-0a53ff6f5d70.png">

### Context
In https://github.com/Automattic/wp-calypso/pull/55791 we've implemented the mobile version of the welcome tour to show the overlay controls all time on mobile based on resolution. But recently, there are a lot of tablets and laptops with touch screens for which displaying the controls on mouseover is not a good option.

> I can confirm the Welcome Tour overlay controls are visible all the time when loading the modal on a wide desktop screen and then resizing the window to mobile resolution.
> 
> Some notes for a possible follow-up:
> 
> * Can we achieve the same using just SCSS?
> * More importantly, can we develop a solution to show these buttons not based on resolution but on the type of screen (touch)?
> * Or maybe should we just show them all the time? @poligilad-auto

_Originally posted by @chriskmnds in https://github.com/Automattic/wp-calypso/issues/55754#issuecomment-907162806_

### A/C

- [ ] Show the overlay control buttons by default
- [ ] Hide the controls until mouse over on devices with hover capabilities and a precision pointer connected. This can be achieved with [a couple of CSS Level 5 media queries](https://www.w3.org/TR/mediaqueries-5/#mf-interaction).
